db/seeds.rb

```diff
@@ -17,3 +17,14 @@
 garden.env_measurements.create!(soil_temperature: 63.88, soil_moisture: 73.90, created_at: 30.minutes.ago)
 garden.env_measurements.create!(soil_temperature: 65.55, soil_moisture: 71.02, created_at: 15.minutes.ago)
 garden.env_measurements.create!(soil_temperature: 69.33, soil_moisture: 68.86)
+
```

Comment:
Thanks for doing this.  I am working on job functionality with the pi currently.  

I think we can get around all this time param filtering with web sockets.